src/types/input.ts

```diff
@@ -51,9 +52,7 @@ export type InputActionsProps = {
 };
 
 export type ExtraAction = {
-    icon: ReactElement;
-    title: string;
-    callback: () => void;
+    tooltip: TooltipProps;
```

Comment:
Are we enforcing the use of Tooltip or just provide the option to add one if we want to?
Content on the Tooltip is required, I'm wondering what would be usage guidelines here => Actions buttons should always have a tooltip?

Reply:
Currently the structure is that any `extraAction` is a `Tooltip` with any `child` element they wish to use.  This is what is reflected in the Figma file.  But can also modify this to allow for more flexibility.

Reply:
I see. Then that's fine but we could anyway double-check and get confirmation with the Designers.

Reply:
OK will raise it at today's sync then can proceed from there.

Reply:
@imoutaharik I have made the updates to allow a tooltip or element.  Interested in your thoughts on the implementation.